index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <main>
+      <section id="form-body">
+        <div class="section-heading">
+          <h1 id="title">freeCodeCamp Survey Form</h1>
+          <p id="description">Thank you for taking the time to help us improve the platform</p>
+        </div>
+        <section class="form-box">
+          <form action="#" id="survey-form">
+            <div class="fields">
+              <label for="" id="name-label">Name</label>
+              <input type="text" id="name" required placeholder="Enter your name">
+            </div>
+            <div class="fields">
+              <label for="" id="email-label">Email</label>
+              <input type="text" id="email" required placeholder="Enter your email">
+            </div>
+            <div class="fields">
+              <label for="" id="number-label">Age</label>
+              <input type="number" min = "10" max=" 70" required placeholder="Age">
+            </div>
+            <div class="fields">
+              <h3 for="">Which option best describes your current role?</>
+              <select name="dropdown" id="dropdown" required>
+                <option value="" disabled>Select current role</option>
+                <option value="Student">Student</option>
+                <option value="Fulltime Job">Fulltime Job</option>
+                <option value="Fulltime Learner">Fulltime Learner</option>
+                <option value="Prefer Not to say">Prefer Not to say</option>
+                <option value="Other">Other</option>
+              </select>
+            </div>
+            <div class="fields" >
+              <h3 for="">Would you recommend freeCodeCamp to a friend?</h3>
+              <input type="radio" name="Definitely" required>
+              <label for="">Definitely</label>
+              <input type="radio" name="Maybe">
+              <label for="">Maybe</label>
+              <input type="radio" name="Not Sure">
+              <label for="">Not Sure</label>
+            </div>
+            <div class="fields">
+              <h3 for="">What is your favorite feature of freeCodeCamp?</>
+              <select name="dropdown" id="dropdown" required>
+                <option value="" disabled>Select an Option</option>
+                <option value="Student">Challenges</option>
+                <option value="Fulltime Job">Projects</option>
+                <option value="Fulltime Learner">Community</option>
+                <option value="Prefer Not to say">Open Source</option>
+              </select>
+            </div>
+            <div class="fields">
+              <h3>What would you like to see improved? (Check all that apply)</h3>
+              <input type="checkbox" id="Front-end Projects" name="Front-end Projects" value="Front-end Projects">
```

Comment:
Kindly apply flex properties to the checkbox element so that the text and the box are aligned in the same line and give the elements the same amount of margin to maintain consistency
![screenshot](https://user-images.githubusercontent.com/29442846/185892789-6ba5afe1-a913-42fa-b7ef-acc55ddf4a62.png)
.